deploy/crds/app.stacks_runtimecomponents_crd.yaml

```diff
@@ -813,8 +813,8 @@ spec:
                             type: string
                           port:
                             anyOf:
-                            - type: integer
```

Comment:
Please rerun `make generate` and then commit the changes for CRD, `*openapi.go` and `*deepcopy.go`

Reply:
I don't see the `service.portName` changes in CRD and also the other two `*.go` files